requirements.txt

```diff
@@ -1,23 +1,23 @@
 coala==0.4.2.dev20160316074840
```

Comment:
coala too
@sils1297 how do we handle this ? We first move coala to 0.5.0 and then change this to ~0.5.0 ?